docs/modules/user-guide/partials/proc_updating-the-devfile-library-schema.adoc

```diff
@@ -0,0 +1,13 @@
+[id="proc_updating-the-devfile-library-schema_{context}"]
+= Updating the devfile library schema
+
+Update the schema version to ensure that you develop with the latest code from devfile properties:
+
+. To fetch the latest `github.com/devfile/api` Go module and to update the schema saved under `pkg/devfile/parser/data`, run, `./scripts/updateApi.sh`.
+. Update the devile schema version by running, `./updateAPI.sh 2.x.x`.
```

Comment:
```suggestion
. Update the devile schema version by running `./updateAPI.sh 2.x.x`.
```
I don't think a comma is required before an inline CLI command

Reply:
Can we also make this easier for users to copy?